crates/rune-modules/src/http.rs

```diff
@@ -192,6 +1243,69 @@ impl StatusCode {
         rune::vm_write!(f, "{}", self.inner);
         VmResult::Ok(())
     }
+
+    /// Returns the `Integer` corresponding to this `StatusCode`.
+    #[rune::function(instance)]
+    #[inline]
+    fn as_integer(&self) -> i64 {
+        self.inner.as_u16().into()
+    }
```

Comment:
By default, it would be `u16`, but since Rune uses `i64`, I opted for `Integer`.

Reply:
Alright, I've reverted to using `as_u16`.